lib/dor/item.rb

```diff
@@ -19,7 +19,7 @@ def members
       unless @members # if members have not been fetched and cached for this object yet, fetch them
 
         handler = proc do |exception, attempt_number, _total_delay|
-          LyberCore::Log.debug "#{exception.class} on dor-fetcher-service get members call #{attempt_number} for #{@druid}" if attempt_number >= Dor::Config.release.max_tries
+          raise exception if attempt_number >= Dor::Config.release.max_tries
```

Comment:
Do we still need the handler at all? I had assumed this behavior was implied by `with_retries`?